api/src/main/java/org/openmrs/api/PatientService.java

```diff
@@ -844,4 +845,13 @@ public void saveCauseOfDeathObs(Patient patient, Date dateDied, Concept causeOfD
 	 * @throws PatientIdentifierTypeLockedException
 	 */
 	public void checkIfPatientIdentifierTypesAreLocked() throws PatientIdentifierTypeLockedException;
-}
\ No newline at end of file
+
+	/**
+	 * Get all patientIdentifiers that match all of the given criteria Voided identifiers are not
+	 * returned
+	 * @param patientProgram the patients containing these identifiers
```

Comment:
@slubwama  is it supposed to be **patients containing these identifiers** or **program containing these identifiers** ?

Reply:
@dkayiwa done with review